### PR TITLE
feat(style): remove usage of kongponents css custom props [KHCP-8877]

### PR DESCRIPTION
**Pull request by adamdehaven**
## Summary

Remove the usage and import of the `@kongponents-css-variables` mixin.

## Ready-To-Review Checklist

- [ ] **Tests:** Includes any new/updated component tests
- [ ] **Docs:** updates [documentation](https://github.com/Kong/khcp/tree/master/packages/docs) as needed
- [ ] **Commit format/atomicity:** the commits follow the guidelines [outlined here](https://github.com/Kong/kong-ee/blob/next/2.1.x.x/CONTRIBUTING.md#commit-atomicity)

## Ready-To-Merge Checklist

- [ ] **Reviewer** - At least one reviewer has reviewed the following:
  - [ ] **Functional:** reviewed acceptance criteria and functionally tested the changes
  - [ ] **Tests:** Reviewer has checked the automated tests for correctness and completeness

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
     "@digitalroute/cz-conventional-changelog-for-jira": "^8.0.1",
     "@evilmartians/lefthook": "^1.4.10",
     "@kong/design-tokens": "^1.10.2",
-    "@kong/kongponents": "^8.123.6",
+    "@kong/kongponents": "^9.0.0-pr.1725.1d9bb913.0",
```
Comment (portikM) [inline package.json:53]: This is pointing to PR preview - is that intentional?

File: README.md
Comment (portikM) [inline README.md:1]: We are referencing old color variables in this file on lines 159 and 279 - can you update those plz?

Comment (adamdehaven) [reply README.md:1]: Removed

File: package.json (hunk 1 above)
Comment (adamdehaven) [reply package.json:53]: Not planning to merge yet - converting back to a draft 👍🏼

**Review by portikM — APPROVED** [review]
LGTM 🚀

**Pull request merged**